packages/hyperdrive-js-core/src/hyperdrive/base/ReadHyperdrive.ts

```diff
@@ -230,24 +228,15 @@ export class ReadHyperdrive extends ReadModel {
     const { vaultSharePrice: currentVaultSharePrice } =
       await this.getPoolInfo(options);
 
-    const timeRange = currentBlock.timestamp - startBlock.timestamp; // bigint
+    const timeFrame = BigInt(currentBlock.timestamp - startBlock.timestamp);
```

Comment:
These should already be bigints, so we can remove the constructor:
```suggestion
    const timeFrame = currentBlock.timestamp - startBlock.timestamp;
```

Reply:
Ah, good catch! my IDE was playing games on me. Fixed it here and on the lpApy function.